schemas/Error.json

```diff
@@ -32,8 +32,20 @@
       "description": "Context - client defined, can be used for debugging.",
       "type": "string"
     },
+    "timeToDecryptMillis": {
+      "description": "UTDs can be permanent or temporary. If temporary, this field will contain the time it took to decrypt the message in milliseconds. If permanent should be -1",
+      "type": "integer"
+    },
+    "cryptoSDK": {
+      "description": "Which crypto backend is the client currently using.",
+      "type": "string",
+      "oneOf": [
+        {"const": "Legacy", "description": "Legacy crypto backend specific to each platform."},
+        {"const": "Rust", "description": "Cross-platform crypto backend written in Rust."}
+      ]
+    },
     "cryptoModule": {
-      "description": "Which crypto module is the client currently using.",
+      "description": "DEPRECATED: Which crypto module is the client currently using.",
```

Comment:
What is the point to deprecate `cryptoModule`? It looks quite similar to `cryptoSDK`, and this will break stats history I think.

Reply:
Web is using cryptoSDK, just had to use the same thing everywhere. cryptoSDK won